Source/common/SNTDeviceEvent.m

```diff
@@ -0,0 +1,54 @@
+#import "Source/common/SNTDeviceEvent.h"
+
+@implementation SNTDeviceEvent
```

Comment:
Would strongly recommend adding a custom init function that takes all of the properties as arguments so as to avoid partially initialized objects. Looks like the only current allocation of this object is down in `SNTDeviceManager.mm` and there isn't doesn't appear to be a reason for partial initialization.

Reply:
Done.